installers/metrics/ansible/roles/pgo-metrics/templates/prometheus-deployment.json.j2

```diff
@@ -26,14 +29,15 @@
 {% endif %}
 {% if not (disable_fsgroup | default(false) | bool) %}
                     {% if (prometheus_supplemental_groups | default('')) != '' %},{% endif -%}
-                    "fsGroup": 26
+                    "fsGroup": 26,
```

Comment:
Do we need `fsGroup` here? AFAIK this is not being run/owned by the `postgres` user.

Reply:
We still need `fsGroup` when using cloud storage, e.g. when running in GKE.  I stuck with `26` since this is what we were using with our old `crunchy` metrics containers and installers and wanted to ensure consistency:

https://github.com/CrunchyData/postgres-operator/blob/6fbd804b57ab68cd4286473ce22b2b1ccf58c25e/installers/ansible/roles/pgo-metrics/templates/grafana-deployment.json.j2#L29

Since Kubernetes adds the user running the container to the `fsGroup` specified, and then modifies the group ownership of the files with that same group, this allows proper access to the volumes, even though there is a different user running the container (so with this PR `fsGroups` and users essentially remain consistent with how they previously were with the old metrics containers & installers).

Reply:
OK.